src/main/java/com/uber/cadence/internal/compatibility/Thrift2ProtoAdapter.java

```diff
@@ -0,0 +1,899 @@
+      ServiceBusyError, ClientVersionNotSupportedError, TException {
+    com.uber.cadence.api.v1.QueryWorkflowResponse response =
+        grpcServiceStubs
+            .workflowBlockingStub()
+            .queryWorkflow(
+                RequestMapper.queryWorkflowRequest(queryRequest));
+    return ResponseMapper.queryWorkflowResponse(response);
+  }
+
+  @Override
+  public DescribeWorkflowExecutionResponse DescribeWorkflowExecution(
+      DescribeWorkflowExecutionRequest describeRequest)
+      throws BadRequestError, EntityNotExistsError, LimitExceededError, ServiceBusyError,
+      ClientVersionNotSupportedError, TException {
+    com.uber.cadence.api.v1.DescribeWorkflowExecutionResponse response =
+        grpcServiceStubs
+            .workflowBlockingStub()
+            .describeWorkflowExecution(
+                RequestMapper.describeWorkflowExecutionRequest(describeRequest));
+    return ResponseMapper.describeWorkflowExecutionResponse(response);
+  }
+
+  @Override
+  public DescribeTaskListResponse DescribeTaskList(DescribeTaskListRequest request)
+      throws BadRequestError, EntityNotExistsError, LimitExceededError, ServiceBusyError,
+      ClientVersionNotSupportedError, TException {
+    com.uber.cadence.api.v1.DescribeTaskListResponse response =
+        grpcServiceStubs
+            .workflowBlockingStub()
+            .describeTaskList(
+                RequestMapper.describeTaskListRequest(request));
+    return ResponseMapper.describeTaskListResponse(response);
+  }
+
+  @Override
+  public ClusterInfo GetClusterInfo() throws InternalServiceError, ServiceBusyError, TException {
+    com.uber.cadence.api.v1.GetClusterInfoResponse response =
+        grpcServiceStubs
+            .workflowBlockingStub()
+            .getClusterInfo(com.uber.cadence.api.v1.GetClusterInfoRequest.newBuilder().build());
+    return ResponseMapper.getClusterInfoResponse(response);
+  }
+
+  @Override
+  public GetTaskListsByDomainResponse GetTaskListsByDomain(GetTaskListsByDomainRequest request) {
+    throw new UnsupportedOperationException("not implemented");
+  }
+
+  @Override
+  public ListTaskListPartitionsResponse ListTaskListPartitions(
+      ListTaskListPartitionsRequest request)
+      throws BadRequestError, EntityNotExistsError, LimitExceededError, ServiceBusyError,
+      TException {
+    com.uber.cadence.api.v1.ListTaskListPartitionsResponse response =
+        grpcServiceStubs
+            .workflowBlockingStub()
+            .listTaskListPartitions(
+                RequestMapper.listTaskListPartitionsRequest(request));
+    return ResponseMapper.listTaskListPartitionsResponse(response);
+  }
+
+  @Override
+  public void RefreshWorkflowTasks(RefreshWorkflowTasksRequest request)
+      throws BadRequestError, DomainNotActiveError, ServiceBusyError, EntityNotExistsError,
+      TException {
+    throw new UnsupportedOperationException("not implemented");
+  }
```

Comment:
I assume this is going to implemented in a follow up PR?

Reply:
yes, the most of the methods are not used at all so we will follow up as we did for Go SDK